controllers/message-controller.ts

```diff
@@ -39,130 +37,17 @@ export default class MessageController implements MessageControllerI {
         if(MessageController.messageController === null) {
             MessageController.messageController = new MessageController();
 
-            app.post("/api/users/:uid1/messages/:uid2", MessageController.messageController.userSendsMessage);
-            app.get("/api/users/:uid/messages", MessageController.messageController.findAllMessagesSentByUser);
-            app.get("/api/users/:uid/received-messages", MessageController.messageController.findAllMessagesReceivedByUser);
-            app.delete("/api/messages/:mid", MessageController.messageController.userDeletesMessage);
-            app.put("/api/messages/:mid", MessageController.messageController.userUpdatesMessage);
-            app.delete("/api/users/:uid/delete-sent-messages", MessageController.messageController.deleteAllMessagesSentByUser);
-            app.delete("/api/users/:uid/delete-received-messages", MessageController.messageController.deleteAllMessagesReceivedByUser);
-            app.get("/api/users/:uid1/messages/:uid2", MessageController.messageController.findAllMessagesBetweenSpecificUsers);
-            app.get("/api/messages/:mid", MessageController.messageController.findMessageById);
-            app.get("/api/messages", MessageController.messageController.findAllMessages);
+            app.post("/api/users/:uid1/messages/:uid2", MessageController.messageService.userSendsMessage);
+            app.delete("/api/messages/:mid", MessageController.messageService.userDeletesMessage);
+            app.put("/api/messages/:mid", MessageController.messageService.userUpdatesMessage);
+            app.get("/api/users/:uid1/messages/:uid2", MessageController.messageService.findAllMessagesBetweenSpecificUsers);
+            app.get("/api/messages/:mid", MessageController.messageService.findMessageById);
+            app.get("/api/messages", MessageController.messageService.findAllMessages);
```

Comment:
I think in one of the meetings Shubham suggested not to have findAllMessages in our controller. Just to keep in the dao?